src/main/java/jenkins/plugins/logstash/LogstashInstallation.java

```diff
@@ -124,5 +125,15 @@ public FormValidation doCheckString(@QueryParameter("value") String value) {
 
       return FormValidation.ok();
     }
+    public FormValidation doCheckMimeType(@QueryParameter("value") String value) {
+      if (StringUtils.isBlank(value)) {
+            return FormValidation.error(Messages.ValueIsRequired());
+        }
+      // Mime type validation as per RFC-4288.
+      if (!value.matches("^[\\w#&+_$.\\-\\^]+/[\\w#&+_$.\\-\\^]+$")) {
```

Comment:
I think it's better to reuse https://hc.apache.org/httpcomponents-core-ga/httpcore/apidocs/org/apache/http/entity/ContentType.html#parse(java.lang.String)

Reply:
I did think of this earlier. But I did not end up using it in my code because I realized it does not actually parse `Mime-Type`. for e.g. `ContentType.parse("application/json%%")` (which is clearly not a valid mime-type) does not complain instead it returns `ContentType `object with this invalid mimetype. 
For this reason I did not use it in the code and ended up validating by my own as per RFC-4288. This would potentially eliminate unwanted surprises that could cause with invalid mime-type.

Reply:
OK, I assumed 
```
    ParseException - if the given text does not represent a valid Content-Type value.
```
means it dies validate it

Reply:
It just validates Content-Type as a whole and does not look for mime-type validity.

Reply:
You could use javax.activation.MimeType
This does validate the mimetype. Though it seems `&` is a valid character in a mimetype. Mimetype fails when you use `§`
see RFC 2045